caps/types.go

```diff
@@ -90,6 +111,21 @@ func (t *TestType) Sanitize(c *Capability) error {
 	return nil
 }
 
+// SecuritySnippet for test capability type.
+// Consumers don't gain any extra permissions.
+func (t *TestType) SecuritySnippet(c *Capability, securitySystem string) (string, error) {
+	switch securitySystem {
+	case SecurityApparmor:
+		fallthrough
```

Comment:
Changed, let me know what you think. I've opted for simplicity so there's no callback yet. Once we have a need for one it can be easily added.